velox/connectors/hive/TableHandle.h

```diff
@@ -96,6 +98,10 @@ class HiveColumnHandle : public ColumnHandle {
     return columnType_ == ColumnType::kPartitionKey;
   }
 
+  const std::unordered_map<std::string, std::string>& columnParameters() const {
+    return columnParameters_;
+  }
+
   std::string toString() const;
```

Comment:
toString() should also be enhanced to show the columnParameters_.

Please update unit tests as well.